src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,59 @@
+@import '../utils/variables';
+
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    color: $normal-color;
+  }
+
+  &__item {
+    margin: 0 10px;
+  }
+
+  &__item:first-child {
+    margin-left: 0;
+  }
+
+  &__item:last-child {
+    margin-right: 0;
+  }
+
+  &__link {
+    position: relative;
+    display: flex;
+    align-items: center;
+    height: 60px;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: $normal-color;
+
+    &:hover {
+      color: $active-color;
+    }
+
+    &.is-active {
+      color: $active-color;
+    }
+  }
+}
+
+.is-active {
+  color: $active-color;
```

Comment:
Ensure that `$active-color` is defined in the imported variables file. If not, this could lead to a compilation error or unexpected styling.